pkg/k8s/types.go

```diff
@@ -0,0 +1,10 @@
+package k8s
+
+const (
+	// ApplicationCrdName defines the CRD name for Application of application-connector module.
+	ApplicationCrdName string = "applications.applicationconnector.kyma-project.io"
+	// ApplicationKind defines the Kind name for Application of application-connector module.
+	ApplicationKind string = "Application"
+	// ApplicationAPIVersion defines the API version for Application of application-connector module.
+	ApplicationAPIVersion string = "applicationconnector.kyma-project.io/v1alpha1"
```

Comment:
Please check if the Kyma application connector package have similar constants or not, so we can reuse them.

Reply:
Sure